SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorModule.cpp

```diff
@@ -300,22 +279,9 @@ bool FSpatialGDKEditorModule::ForEveryServerWorker(TFunction<void(const FName&,
 
 void FSpatialGDKEditorModule::OverrideSettingsForTesting(UWorld* World, const FString& MapName)
 {
-	// By default, clear that the runtime/test was loaded from a snapshot taken for a given world.
-	ASpatialFunctionalTest::ClearLoadedFromTakenSnapshot();
+	OverrideSettingsForTestingDelegate.Broadcast(World, MapName);
 
 	SpatialTestSettings->Override(MapName);
-
-	if (GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
-	{
-		FString SnapshotForMap = ASpatialFunctionalTest::GetTakenSnapshotPath(World);
-
-		if (!SnapshotForMap.IsEmpty())
-		{
-			GetMutableDefault<ULevelEditorPlaySettings>()->SetSnapshotOverride(SnapshotForMap);
-			// Set that we're loading from taken snapshot.
-			ASpatialFunctionalTest::SetLoadedFromTakenSnapshot();
-		}
-	}
```

Comment:
Where is this functionality now? Is there a corresponding TestGym PR?

Reply:
Very sorry about this, realized I forgot to add two files in git.
The functionality is in `SpatialGDKFunctionalTestsModule.cpp` and `SpatialGDKFunctionalTestsModule.h`.